src/components/Chip.vue

```diff
@@ -0,0 +1,319 @@
+<!-- eslint-disable vue/multi-word-component-names -->
+<template>
+	<div
+		class="chip__container"
+		:class="classList"
+		@click="handleClick"
+	>
+		<div
+			:class="{
+				'chip__content--sm': size === 'sm',
+				'chip__content--md': size === 'md',
+				'chip__content--lg': size === 'lg',
+			}"
+		>
+			<div
+				class="chip__content-container"
+				:style="{
+					maxWidth: maxWidth,

```

Comment:
```suggestion
					maxWidth,

```

Acho que Isso aqui funciona nessa estrutura